web_src/js/webcomponents/overflow-menu.js

```diff
@@ -127,6 +127,26 @@ window.customElements.define('overflow-menu', class extends HTMLElement {
   });
 
   init() {
+    // for horizontal menus where fomantic boldens active items, prevent this bold text from
+    // enlarging the menu's active item replacing the text node with a div that renders a
+    // invisible pseudo-element that enlarges the box.
+    if (this.matches('.ui.secondary.pointing.menu') || this.matches('.ui.tabular.menu')) {
+      for (const item of this.querySelectorAll('.item')) {
+        for (const child of item.childNodes) {
+          if (child.nodeType === Node.TEXT_NODE) {
+            const text = child.textContent.trim();
+            if (text) {
+              const div = document.createElement('div');
+              div.classList.add('size-for-semibold');
```

Comment:
If I understand correctly, the newly created element is: "a element with fixed with for bold and non-bold fonts". So maybe some names like `fixed-width-for-semibold`

Reply:
And one more thing, the newly created element should be `span` (inline element), but not `div` (block element)

Reply:
FYI, The new element renders on top of the old one, here is how it looks with `visibility: hidden` removed:

<img width="485" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/90767c21-c9bb-402f-bbf3-0f87290a5884">

Reply:
TBH I think the class name is fitting, it "sizes the element as if it had semibold text".

Reply:
Hmm, I never thought it is a verb ... actually there is seldom verb in CSS names, for example: `box-sizing`, `padding`, there are all `ing` nouns.

Reply:
I can rename to `resize` if that's more clear.

Reply:
Renamed in https://github.com/go-gitea/gitea/pull/29831/commits/b48978d28437f7b4a0434a710da3e1333300f3a3. "resize" is a more common verb.